jest.config.ts

```diff
@@ -35,19 +33,20 @@ const config: Config.InitialOptions = {
 			branches: 80,
 			functions: 80,
 			lines: 80,
-			statements: -7,
+			statements: -11,
 		},
 		// add custom paths: './apps/server/path...': { branches: X, functions: ... }
 	},
 	testEnvironment: 'node',
 	// detectOpenHandles: true,
 	// detectLeaks: true,
 	roots: ['<rootDir>/apps/'],
+	globalSetup: '<rootDir>/apps/server/test/globalSetup.ts',
```

Comment:
Is this a beforeAllAll and a afterAllAll, or is it is used for each test file, each test?
For nest e2e test and repo integration test we do not need the in memory database.

Reply:
Yes but we have just one jest.config.ts and it is just one instance from mongodb with different dbnames for tests